astropy/coordinates/builtin_frames/cirs_observed_transforms.py

```diff
@@ -40,7 +40,8 @@ def cirs_to_altaz(cirs_coo, altaz_frame):
     xp, yp = get_polar_motion(obstime)
 
     #first set up the astrometry context for CIRS<->AltAz
-    astrom = erfa.apio13(obstime.jd1, obstime.jd2,
+    obstime_utc = obstime if obstime.scale == 'utc' else obstime.utc
```

Comment:
Actually, @eteq - since this seem to happen mostly when you are getting out `jd1` and `jd2`, maybe this would be something for a very small helper function instead:
```
def jds(time, scale=None):
    t = time if scale is None or scale == time.scale else getattr(time, scale)
    return t.jd1, t.jd2
```
which allows you do do:
```
jd1, jd2 = jds(obstime, scale='utc')
```
(or for simple functions to put in directly as arguments: `erfa.obl06(*jds(obstime, 'tt'))`)